inc/class-wpseo-replace-vars.php

```diff
@@ -1146,15 +1146,85 @@ private function get_replacement_variables() {
 	 *
 	 * @param string $replacement_variable The replacement variable.
 	 *
-	 * @return bool True when the replacement variable contains a
+	 * @return bool True when the replacement variable is not prefixed.
 	 */
 	private function is_not_prefixed( $replacement_variable ) {
-		$prefixes = array( 'cf_', 'ct_' );
-		$prefix   = substr( $replacement_variable, 0, 3 );
+		$prefixes = array( 'cf_', 'ct_', 'pt_' );
+		$prefix   = $this->get_prefix( $replacement_variable );
 
 		return ! in_array( $prefix, $prefixes );
 	}
 
+	/**
+	 * Strip the prefix from a replacement variable name.
+	 *
+	 * @param string $replacement_variable The replacement variable.
+	 *
+	 * @return string The replacement variable name without the prefix.
+	 */
+	private function strip_prefix( $replacement_variable ) {
+		return substr( $replacement_variable, 3 );
+	}
+
+	/**
+	 * Gets the prefix from a replacement variable name.
+	 *
+	 * @param string $replacement_variable The replacement variable.
+	 *
+	 * @return string The prefix of the replacement variable.
+	 */
+	private function get_prefix( $replacement_variable ) {
+		return substr( $replacement_variable, 0, 3 );
+	}
+
+	/**
+	 * Strips 'desc_' if present, and appends '(description)' at the end.
+	 *
+	 * @param string $label The replacement variable.
+	 *
+	 * @return string The altered replacement variable name.
+	 */
+	private function handle_description( $label ) {
+		if ( substr( $label, 0, 5 ) === 'desc_' ) {
+			return substr( $label, 5 ) . ' description';
+		}
+		return $label;
+	}
+
+	/**
+	 * Creates a label for prefixed replacement variables that matches the format in the editors.
+	 *
+	 * @param string $replacement_variable The replacement variable.
+	 *
+	 * @return string The replacement variable label.
+	 */
+	private function get_prefix_label( $replacement_variable ) {
+		$label = '';
+		$prefix = $this->get_prefix( $replacement_variable );
+		switch ( $prefix ) {
```

Comment:
To reduce complexity I would choose for replacing this `switch` by three if-statements.